docs/src/dispersion.md

```diff
@@ -0,0 +1,139 @@
+# Dispersive properties of the equations
+
+The equations implemented in [DispersiveShallowWater.jl](https://github.com/JoshuaLampert/DispersiveShallowWater.jl)
+describe the propagation of waves in a shallow water system.
+The equations are dispersive, meaning that the phase speed of the waves depends on their wavelength. This is in
+contrast to non-dispersive waves, where all waves travel at the same speed, such as the solution to the shallow
+water equations linearized around a state with vanishing velocity and constant water height ``h_0``.
+In this case the phase speed is constantly given by ``c_0 = \sqrt{g h_0}``, where ``g`` is the
+acceleration due to gravity and ``h_0`` is a reference water height.
+
+The linear dispersion relation of a wave equation describes the relationship between the angular frequency
+``\omega`` and the wavenumber ``k`` of a wave. The angular frequency is related to the period ``T`` of the wave
+by ``\omega = 2\pi / T`` and the wavenumber is related to the wavelength ``\lambda`` by ``k = 2\pi / \lambda``.
+The simple case of just one wave traveling with speed ``c`` is a harmonic (or plane wave) solution of the form
+``\eta(x, t) = \mathrm{e}^{\mathrm{i}(k x - \omega t)}`` with ``c = \omega / k``. The linear dispersion relation
```

Comment:
```suggestion
``\eta(t, x) = \mathrm{e}^{\mathrm{i}(k x - \omega t)}`` with ``c = \omega / k``. The linear dispersion relation
```

Is there a reason to prefer the order $x, t$ instead of $t, x$? I often use $t$ first since it appears to be more natural when looking at semidiscretizations and writing stuff like $u(t) \in L^2$ etc.

Reply:
I think the only reason was that `x` comes before `t` in the initial conditions (same as in Trixi.jl). But I'm fine with using t first.